Tests/DependencyInjection/Compiler/ResolverTaggedServiceMappingPassTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Overblog\GraphQLBundle\Tests\DependencyInjection\Compiler;
+
+use Overblog\GraphQLBundle\DependencyInjection\Compiler\ResolverTaggedServiceMappingPass;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Class ResolverTaggedServiceMappingPassTest
+ * @package DependencyInjection
+ */
+class ResolverTaggedServiceMappingPassTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var ContainerBuilder
+     */
+    private $container;
+
+    public function setUp()
+    {
+        $container = new ContainerBuilder();
+        $container->setDefinition(
+            'injected_service',
+            new Definition('Overblog\GraphQLBundle\Tests\DependencyInjection\Compiler\FakeInjectedService')
+        );
+
+        $container->register(
+            'overblog_graphql.resolver_resolver',
+            "Overblog\\GraphQLBundle\\Resolver\\ResolverResolver"
+        );
+
+        $fakeResolver = new Definition('Overblog\GraphQLBundle\Tests\DependencyInjection\Compiler\FakeResolverService');
+        $fakeResolver
+            ->addTag( 'overblog_graphql.resolver', [
+                'alias' => 'fake_resolver', 'method' => 'doSomethingWithContainer'
+            ]);
+
+        $container->setDefinition('fake_resolver', $fakeResolver);
+
+        $this->container = $container;
+    }
+
+    private function addCompilerPassesAndCompile()
+    {
+            $this->container->addCompilerPass(new ResolverTaggedServiceMappingPass());
+            // Manipulate container after ResolverTaggedServiceMappingPass
+            $this->container->addCompilerPass(new FakeCompilerPass());
+            $this->container->compile();
+    }
+
+    public function testCompilationWorksPassConfigDirective()
+    {
+        $this->addCompilerPassesAndCompile();
+
+        $this->assertTrue($this->container->has('fake_resolver'));
+    }
+}
+
+class FakeCompilerPass implements CompilerPassInterface
```

Comment:
Create separate files for each class to respect  psr4 please, thanks for the work :)